### ISSUE TITLE
tooltips in ofParameterGroup

### ISSUE THREAD

**Issue by stephanschulz** [roymacdonald/ofxGuiTooltip#2]
I see that the `example-json` loads in parameters for `ofParameterGroup` elements.

Do you have an example on how to do that inline?
This seems not to work ` tooltips.add(&group_autoFake, maxLocalActiveDuration, "max wait time before faking a response");`

**Comment by roymacdonald** [comment]
Hi Stephan,
Does the example-inline work? It works for me.
What is that you want to achieve that you cant?

**Comment by stephanschulz** [comment]
I noticed what my mistake was. 
I added the tooltips right after adding the group elements. 
But your example adds the tooltips after the group gets added to the gui.

So this does not work
```
    gui.setup("GUI");
    groups.setName("Group 1");
    tooltips.add(&gui, groups,"this is the group tooltip");
    groups.add(floats1);
    tooltips.add(&gui, floats1, "Tooltip for floats1");
    groups.add(floats2);
    tooltips.add(&gui, floats2, "Tooltip for floats2");
    
    gui.add(groups);
```


But this works
```
    gui.setup("GUI");
    groups.setName("Group 1");
    groups.add(floats1);
    groups.add(floats2);
   
    gui.add(groups);
    
    tooltips.add(&gui, groups,"this is the group tooltip");
    tooltips.add(&gui, floats1, "Tooltip for floats1");
    tooltips.add(&gui, floats2, "Tooltip for floats2");
```

I was hoping that right after adding the group elements I could add the tooltips for more clarity.

**Comment by roymacdonald** [comment]
Right, it has to be that way because my addon has to find the gui element related to be the ofParameter, but the gui elements only get created when you add the ofParameter or ofParameterGroup to the gui. If instead of using ofParameters you directly instatiate gui elements (ofxSlider, etc) you can setup the tooltip before adding the gui element to the gui. Im the inline example there are 2 sliders, which you register in a different way than the ofParameters
